requirements.txt

```diff
@@ -1 +1,2 @@
+python-dotenv
```

Comment:
should we use file? i think environment variable is enough.
@evgeniy-tulyakov what are your arguments?

Reply:
creating a ".env" file  is not a requirement. 
The bot loads this file only if it is created. 
creating this file absolvs us from permanent use of the "export" command.